change/@ni-nimble-components-297ab22e-342a-48cb-94f9-a5d8e7fe9bfa.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Finished first iteration of the wafermap component and its storybook",
+  "packageName": "@ni/nimble-components",
+  "email": "denis.stavila@ni.com",
```

Comment:
In my PR I got this comment, maybe will help you too :) 
"In [the root CONTRIBUTING.md](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#beachball-change-file) there are instructions about how to configure GitHub to use a private email address to avoid exposing your NI address in public. For this PR you can manually modify this JSON file to copy in the address that GitHub shows at https://github.com/settings/emails"

https://github.com/ni/nimble/pull/818#discussion_r1021708842

Reply:
Done ... event though I am still not sure how it works - I did have this setting already set
![image](https://user-images.githubusercontent.com/35329597/203812620-efe02c70-9c09-4027-9639-38db8d5d896a.png)